charts/tensorleap/templates/rabbitmq-cm.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rabbitmq-config
+data:
+  enabled_plugins: |
+    [rabbitmq_management].
+  rabbitmq.conf: |
+    log.file.level = debug
+    log.console = true
+    log.console.level = debug
+  erlang_cookie: cookie123
```

I think this will be more similar to what we had before (check me)
```suggestion
  erlang_cookie: 3e3abae2-6325-11ec-90d6-0242ac120003
    erlangCookie: 
```
Also, remove the `rabbitmq` section from `values.yaml`

---

Done & done